tests/test_gpu_examples.py

```diff
@@ -2722,10 +2722,8 @@ def test_hqq_lora_model_outputs(self):
         assert cc_matrix.min() > 0.97
 
 
-# TODO: unskip the tests once https://github.com/casper-hansen/AutoAWQ/issues/466 is fixed
```

Comment:
Unrelated?

Reply:
Not totally unrelated, I had to update the AutoAWQ wheels and the new wheels should contain the fix mentioned here.